activitysim/examples/prototype_mtc_extended/configs/school_escorting_inbound.csv

```diff
@@ -0,0 +1,175 @@
+util_chauf2_female_pe,Chauffeur 2 Female - Pure Escort,(gender_chauf2 == 2) & ((chauf1 == 4) | (chauf2 == 4) | (chauf3 == 4)),coef_chauf_female_pe
+util_chauf2_male_pe,Chauffeur 2 Male - Pure Escort,(gender_chauf2 == 1) & ((chauf1 == 4) | (chauf2 == 4) | (chauf3 == 4)),coef_chauf_male_pe
+util_chauf1_pt_worker_rs,Chauffer 1 part time worker - Ride share,(ptype_chauf1 == 2) & ((chauf1 == 1) | (chauf2 == 1) | (chauf3 == 1)),coef_chauf_pt_worker_rs
+util_chauf1_pt_worker_pe,Chauffer 1 part time worker - Pure escort,(ptype_chauf1 == 2) & ((chauf1 == 2) | (chauf2 == 2) | (chauf3 == 2)),coef_chauf_pt_worker_pe
+util_chauf2_pt_worker_rs,Chauffer 2 part time worker - Ride share,(ptype_chauf2 == 2) & ((chauf1 == 3) | (chauf2 == 3) | (chauf3 == 3)),coef_chauf_pt_worker_rs
+util_chauf2_pt_worker_pe,Chauffer 2 part time worker - Pure escort,(ptype_chauf2 == 2) & ((chauf1 == 4) | (chauf2 == 4) | (chauf3 == 4)),coef_chauf_pt_worker_pe
+util_chauf1_non_worker_pe,Chauffer 1 non worker - Pure escort,(ptype_chauf1 == 4) & ((chauf1 == 2) | (chauf2 == 2) | (chauf3 == 2)),coef_chauf_non_worker_pe
+util_chauf2_non_worker_pe,Chauffer 2 non worker - Pure escort,(ptype_chauf2 == 4) & ((chauf1 == 4) | (chauf2 == 4) | (chauf3 == 4)),coef_chauf_non_worker_pe
+util_chauf1_univ_stud_re,Chauffer 1 university student - Ride Share,(ptype_chauf1 == 3) & ((chauf1 == 1) | (chauf2 == 1) | (chauf3 == 1)),coef_chauf_univ_stud_re
+util_chauf2_univ_stud_re,Chauffer 2 university student - Ride Share,(ptype_chauf2 == 3) & ((chauf1 == 3) | (chauf2 == 3) | (chauf3 == 3)),coef_chauf_univ_stud_re
+util_chauf1_age_u35_pe,Chauffer 1 Age 35 years or younger - Pure escort,(ptype_chauf1 == 4) & ((chauf1 == 2) | (chauf2 == 2) | (chauf3 == 2)),coef_chauf_age_u35_pe
+util_chauf2_age_u35_pe,Chauffer 2 Age 35 years or younger - Pure escort,(ptype_chauf2 == 4) & ((chauf1 == 4) | (chauf2 == 4) | (chauf3 == 4)),coef_chauf_age_u35_pe
+util_chauf1_time_to_work_or_univ_rs,Chauffer 1 Auto time to work or university - Ride Share,time_mand_to_home1 * ((chauf1 == 1) | (chauf2 == 1) | (chauf3 == 1)),coef_chauf_time_to_work_or_univ_rs
+util_chauf2_time_to_work_or_univ_rs,Chauffer 2 Auto time to work or university - Ride Share,time_mand_to_home2 * ((chauf1 == 3) | (chauf2 == 3) | (chauf3 == 3)),coef_chauf_time_to_work_or_univ_rs
+util_chauf1_walk_dist_to_work_or_univ_rs,Chauffer 1 Walk dist to work or university - Ride Share,(dist_mand1_to_home < 3) & ((chauf1 == 1) | (chauf2 == 1) | (chauf3 == 1)),coef_chauf_walk_dist_to_work_or_univ_rs
+util_chauf2_walk_dist_to_work_or_univ_rs,Chauffer 2 Walk dist to work or university - Ride Share,(dist_mand2_to_home < 3) & ((chauf1 == 3) | (chauf2 == 3) | (chauf3 == 3)),coef_chauf_walk_dist_to_work_or_univ_rs
+util_chauf1_walk_dist_to_work_or_univ_pe,Chauffer 1 Walk dist to work or university - Pure Escort,(dist_home1_to_mand < 3) & ((chauf1 == 2) | (chauf2 == 2) | (chauf3 == 2)),coef_chauf_walk_dist_to_work_or_univ_pe
```

Comment:
There's an error in the escorting spec here. `dist_home1_to_mand` isn't a valid variable, it should be `dist_home_to_mand1`.  And again on the following line for "2".  When running in normal simulation mode the error is avoided and our test can pass this module, as the coefficient on this line happens to be set to 0 in the associated coefficients file.  But this will cause errors if run in estimation mode or with sharrow enabled, neither of which is currently tested for the school escorting model.

Reply:
Good catch.  Fixed.  It looks like the outbound and conditional outbound spec files did not have this error.